frontend/app/src/store/utils.ts

```diff
@@ -36,3 +36,11 @@ export const setStatus: (
   };
   commit('setStatus', payload, { root: true });
 };
+
+export function isLoading(status: Status): boolean {
+  return (
+    status === Status.LOADING ||
+    status === Status.PARTIALLY_LOADED ||
+    status == Status.REFRESHING
```

Comment:
Why the differences in comparisons operators between this line and the two above?

Reply:
Because I copy/pasted it and the last time I missed that :D nice catch :)